api/types/transfer/import.proto

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
file name needs to be changed as we have both import and export messages being defined here.